RadixWallet/RadixConnect/RadixConnectModels/Types.swift

```diff
@@ -7,3 +7,29 @@ public typealias PeerConnectionID = Tagged<PeerConnectionIdTag, String>
 
 // MARK: Sendable
 extension PeerConnectionID: Sendable {}
+
+// TODO: move to Sargon
+extension RadixConnectPassword {
+	/// Represents the message to be signed and sent to CE.
+	/// CE uses the same logic to compute its own message.
+	var messageToHash: Data {
+		let prefix = Data("L".utf8)
+		let passwordData = self.value.data
+		var messageData = Data()
```

Comment:
I think this `messageToHash` really ought to be moved into Sargon. Typically this protocol implementations ought to have one shared impl and not duplicated across Kotlin and Swift. Hmm. I think it worth the extra 2h (with CI and then CD time etc) tops it will take.

Reply:
Yes, agree.